### ISSUE TITLE
alternative_names should have `name` not `company_name`

### ISSUE THREAD

**Issue by jpmckinney** [openc/openc-schema#41]
Presently, this produces a `person` with a `company_name` under `alternative_names`.

**Comment by CountCulture** [comment]
This will require changing underlying code in core OC app, and also bots which currently produce/store data as company_name (I agree with the sentiment, however -- it's just the way it is for legacy reasons)

**Comment by jpmckinney** [comment]
Yeah, just creating the issue so that it can be fixed at some much later date.
